less/components/jtable.less

```diff
@@ -0,0 +1,96 @@
+//--------------------------------------------------------------------------
+//
+//  JTable and related components
+//
+//--------------------------------------------------------------------------
+
+//--------------------------------------
+//  Vars / Mixins
+//--------------------------------------
+
+@table-left-right-padding: 24px;
+@table-column-spacing: 32px;
+// !!!IMPORTANT!!! Don't change the above without changing the constants in JTable.jsx to match!
+
+// FIXME: We need to sort out the mess of different colours we've got all over the place
+@table-border-color: #e6eaee;
+@table-header-color: rgba(53, 64, 82, 0.5);
+@table-hover-bg: #fafbfd;
+
+//--------------------------------------
+//  Outer JTable component
+//--------------------------------------
+
+.JTable {
+    border: solid 1px @table-border-color;
+    border-radius: 3px;
+    overflow: hidden;
+
+    display: flex;
+    flex-direction: column;
+
+    &.u-table-maxwidth {
+        max-width: 1170px;
+    }
+}
+
+//--------------------------------------
+//  Row Components
+//--------------------------------------
+
+.JTable-row {
+    display: flex;
+    flex-direction: row;
+    border-top: solid 1px @table-border-color;
+
+    &:first-child {
+        border-top: none;
+    }
+}
+
+a.JTable-row--href {
+    color: inherit;
+
+    &:hover {
+        background: @table-hover-bg;
+        color: inherit;
+    }
+}
+
+//--------------------------------------
+//  Cell Components
+//--------------------------------------
+
+.JTable-cell {
+
+    min-width: 0;
```

Comment:
Good catch. I'm not sure I quite remember why this is in there, so I'll pull it out and see what breaks ;-)